src/boot/bootctl-install.c

```diff
@@ -893,8 +1054,19 @@ int verb_install(int argc, char *argv[], void *userdata) {
                 return 0;
         }
 
-        char *path = strjoina("/EFI/systemd/systemd-boot", arch, ".efi");
-        return install_variables(arg_esp_path, part, pstart, psize, uuid, path, install, graceful);
+        if (!default_loader_path) {
+                log_info("No default loader found. Not changing EFI variables.");
+                return 0;
+        }
+
+        const char* description = "Linux Boot Loader";
+        if (arg_efi_boot_option_description)
+                description = arg_efi_boot_option_description;
+        else if (default_loader_description)
+                description = default_loader_description;
```

Comment:
i think this should remain in pick_efi_boot_option_description(), but add an extra argument for the default loader description param to consider

Reply:
what do you mean here by `pick_efi_boot_option_description()`? I can't see that

Reply:
nvm found now